src/lib/utils/propCategories.js

```diff
@@ -306,6 +305,7 @@ export const COLUMN_ARRAY_NESTED_FUNCTIONS = {
  */
 export const OBJ_MAYBE_FUNCTION_OR_MAP_MAYBE_FUNCTIONS = {
     dataTypeDefinitions: 1,
+    aggFuncs: 1,
```

Comment:
This is really the whole diff??

Reply:
Ummm... yes.

The propCategories is spiderman...